CHANGELOG.md

```diff
@@ -2,11 +2,11 @@
 
 ## 7.4.0
 
-* [FIXED] Allow presence channel authorization to depend on user authentication
+* [ADDED] Allow presence channel authorization to depend on user authentication
```

Comment:
Also, let's add other contributions, such as:
* [CHANGED] Use secure random generator instead of pseudo-random generator [#584](https://github.com/pusher/pusher-js/pull/584)
* [CHANGED] Replace git protocol with HTTPS in gitsubmodules file [#587](https://github.com/pusher/pusher-js/pull/587)